src/commands/init.ts

```diff
@@ -187,19 +192,45 @@ export default class Init extends Command {
     }
     packageJson.scripts = { ...packageJson.scripts, ...scripts };
 
-    if (initializationType === InitType.freshExpress) {
-      packageJson.devDependencies = { ...packageJson.devDependencies, ...packageJsonData.devDependencies };
-    }
+    await this.addDependencies(packageJsonData.dependencies, false, packageJson, flags.projectDir);
+    await this.addDependencies(packageJsonData.devDependencies, true, packageJson, flags.projectDir);
 
     fs.writeFileSync(path.resolve(flags.projectDir, 'package.json'), JSON.stringify(packageJson, null, 2));
+  }
 
+  private async addDependencies(dependencies: string[], isDev: boolean, packageJson: any, projectDir: string) {
+    packageJson.devDependencies = packageJson.devDependencies ? packageJson.devDependencies : {};
+    packageJson.dependencies = packageJson.dependencies ? packageJson.dependencies : {};
+    const promises = dependencies.map(async dependency => {
+      try {
+        const latestVersion = (await execa('npm', ['view', dependency, 'version'], { cwd: projectDir })).stdout;
+        if (isDev) {
+          packageJson.devDependencies[dependency] = `^${latestVersion}`;
```

Comment:
Does this overwrite the dependency versions in the package.json?! or did I misunderstand the code (have not executed it yet)

Reply:
Seems like it overwrites only the versions for jest, jest-junit and sdk-test-utils. That makes more sense... Im still a bit skeptical that this is the best way.

Reply:
I am also not 100% happy with the current solution. The thing is I wanted to acheive the following:
- For a fresh add the latest SDK version and jest to the express framework and make a install all so that the user is directly ready to go (no extra npm i) needed
- For a existing project only add the latest SDK version to the project and install only this.
Since the method was called modifyPackageJson I remove the install and only added the dependencies and then made the install in a seperate step. But in the install step you have to give again the dependencies you want to install. Let's discuss later in person.